backend/src/Squidex.Domain.Apps.Core.Operations/Scripting/Extensions/HttpJintExtension.cs

```diff
@@ -209,4 +235,38 @@ public void Describe(AddDescription describe, ScriptScope scope)
             }
         }
     }
+
+    private static (HttpMethod Method, JsValue Headers, JsValue Body) ParseHttpRequest(JsValue? source)
+    {
+        if (source?.IsObject() != true || source.AsObject() is not ObjectInstance obj)
+        {
+            throw new JavaScriptException("Object is not an object.");
+        }
+
+        if (!obj.TryGetValue("method", out var methodJsValue))
```

Comment:
I think it is a little bit weird to put the method in the init object, but not the URL. So either make required values parameters or put everything in the object